k8s/server-deployment.yaml

```
@@ -0,0 +1,153 @@
+            memory: "512Mi"
+            cpu: "500m"
+        
+        # Security Context
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
+          runAsNonRoot: true
+          runAsUser: 1000
+          capabilities:
+            drop:
+              - ALL
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 12 but found 14

(indentation)